tests/campaigns/test_integration.py

```diff
@@ -410,12 +437,10 @@ def test_downgrades_subscription(self):
 
 class SubscribeToDisplayCampaignCancelledSubDowngrade(
         BaseCancelledSubscriptionTestCase):
-    campaign_type = 'display'
+    campaign_type = CampaignTypes.DISPLAY
 
 
 class TestIterSubscriptions(BaseSubscriptionTestCase):
-    campaign_type = 'search'
```

Comment:
Was it removed for a reason?

Reply:
Sure, we set it as `campaign_type = 'search'` in a base class.

Reply:
thanks! (resolved)